SimpleXML/SimpleXML.php

```diff
@@ -215,6 +215,38 @@ public function __toString () {}
 	 */
 	public function count () {}
 
+    /**
```

Comment:
It turns out that deleting the methods doesn't help, because they are assumed to be present based on the interface. However, marking them as Private appears to have precisely the desired effect.